setup.py

```diff
@@ -1,8 +1,22 @@
-from setuptools import setup
+from setuptools import find_packages, setup
 
-setup(name='stereo_rgb',
-      version='1.0.0',
-      packages=['stereo_rgb'],
+def description():
+      with open('readme.rst') as f:
+            return f.read()
+
+setup(name='terraref-stereo_rgb',
+      version='1.0.4',
+      description='TERRA-REF stereo RGB camera science package',
+      long_description=description(),
+      packages=find_packages(),
+      namespace_packages=['terraref'],
       include_package_data=True,
       url='https://github.com/terraref/stereo_rgb',
+      install_requires=[
```

Comment:
Add classifiers and keywords sections? Should we standardize these, or is there some hierarchy where this is covered in the terraref namespace?

e.g. 

keywords=['field crop', 'phenomics', 'computer vision', 'remote sensing']
'Topic :: Scientific/Engineering :: GIS',